benymng/ubiquitous-fiesta-backend
# Handle session and user permissions

### Changes
- Update user schema to have unique email and uuid as id
- Add supabase provider to be able to access the supabase client
- Create middleware protection for routes
- Create user controller and basic get route

### How To Test
- Start the server and call the `http://localhost:3000/user/` route via postman with and without a bearer token
- You can get the bearer token from the supabase dashboard under the auth.users table
- When you don't have a token, you should get a restricted message but when you have the right one you should be able to see the user credentials of the person that you got the bearer token from.

### Labels


### Additional Notes
